examples/server_profile_template.go

```diff
@@ -0,0 +1,93 @@
+package main
+
+import (
+	"fmt"
+	"github.com/HewlettPackard/oneview-golang/ov"
+	"os"
+)
+
+func main() {
+	var (
+		clientOV                     *ov.OVClient
+		server_profile_template_name = "Test SPT"
+		enclosure_group_name         = "SYN03_EC"
+		server_hardware_type_name    = "SY 480 Gen9 1"
+	)
+	ovc := clientOV.NewOVClient(
+		os.Getenv("ONEVIEW_OV_USER"),
+		os.Getenv("ONEVIEW_OV_PASSWORD"),
+		os.Getenv("ONEVIEW_OV_DOMAIN"),
+		os.Getenv("ONEVIEW_OV_ENDPOINT"),
+		false,
+		800,
+		"*")
+
+	server_hardware_type, err := ovc.GetServerHardwareTypeByName(server_hardware_type_name)
+
+	enc_grp, err := ovc.GetEnclosureGroupByName(enclosure_group_name)
+
+	conn_settings := ov.ConnectionSettings{
+		ManageConnections: true,
+	}
+
+	server_profile_template := ov.ServerProfile{
+		Type:                  "ServerProfileTemplateV5",
+		Name:                  server_profile_template_name,
+		EnclosureGroupURI:     enc_grp.URI,
+		ServerHardwareTypeURI: server_hardware_type.URI,
+		ConnectionSettings:    conn_settings,
```

Comment:
can't see initialScopeUris field in the ServerProfile struct. please add it and verify other fields as well